src/script/hydra-create-user

```diff
@@ -71,6 +92,11 @@ GetOptions("rename-from=s" => \$renameFrom,
 die "$0: one user name required\n" if scalar @ARGV != 1;
 my $userName = $ARGV[0];
 
+my $chosenPasswordOptions = grep { defined($_) }  ($passwordPrompt, $passwordHash, $password);
+if ($chosenPasswordOptions > 1) {
+    die "$0: please specify one of --password-prompt or --password-hash. See --help for more information.\n";
```

Comment:
Maybe this would make it more obvious that you can only use one or the other (or the unmentioned other).

```suggestion
    die "$0: please specify only one of --password-prompt or --password-hash. See --help for more information.\n";
```